backend/src/main/java/com/googleinterns.smb/consumer-backend/SMBKiranaApplication.java

```diff
@@ -8,15 +8,13 @@
 import java.util.Map;
 @RestController
 @SpringBootApplication
-<<<<<<< HEAD:backend/src/main/java/com/googleinterns.smb/consumer-backend/SMBApplication.java
-public class SMBApplication {
-=======
+<<<<<<< HEAD:backend/src/main/java/com/googleinterns.smb/consumer-backend/SMBKiranaApplication.java
```

Comment:
How >>>>>>> is still in the commit
Please remote the git diff lines

Reply:
Done